omniport/core/base_auth/managers/get_user.py

```diff
@@ -49,6 +50,16 @@ def get_user(username):
     except Student.DoesNotExist:
         pass
 
+    try:
+        faculty = FacultyMember.objects.get(employee_id=username)
```

Comment:
Sorry for noticing this so late
wouldn't this cause an error if there is no shell, as the employee_id field doesn't exist in the kernel model.
Maybe some error handling should be handed as well here?

Reply:
That seems to be about talking about a unique field-specific for faculty in the kernel itself, I think Dhruv's suggestion makes sense since other colleges may not be requiring that. 
Just a suggestion, would it make sense if we just use the primary key itself for the faculty_id in shell, and use the id only in case of get_user. Might work if anyone changes the id field in shell and work without it too in kernel

Reply:
We may check the FacultyMember.get_field('employee_id') to check if the field is present and then try to extract the person out of it.